search/app.js

```diff
@@ -14,9 +14,11 @@ const swaggerFileContents = fs.readFileSync('WFS3core+STAC.yaml');
 const swagger = yaml.safeLoad(swaggerFileContents);
 
 const createSchemaValidator = (schemaElement) => {
-  const schema = _.merge({
-    components: swagger.components
-  }, swagger.components.schemas[schemaElement]);
+  // TODO: remove lodash
+  const schema = Object.assign({}, swagger.components.schemas[schemaElement])
```

Comment:
This will not bring in the components object. The merge takes two objects (Maps) and merges the properties of each onto a new object. e.g. merge({a: 1}, {b: 2}) will result in {a: 1, b:2}

Object.assign takes in the same arguments as merge and and a new object. .assign actually mutates the first argument, so it is normally seen as Object.assign({}, objA, objB) where objA and objB are the objects you want to merge and {} is the mutated object that gets returned.

In this case, this line should be:
`const schema = Object.assign({}, { components: swagger.components}, swagger.components.schemas[schemaElement]);`

Reply:
Ok, that makes sense.